From a7093baa088c28402e7ca8dd7aa2b533562eef0e Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Thu, 9 Aug 2018 06:29:27 +0000
Subject: [PATCH] OS-7119 adjust pci slots to support windows Reviewed by:
 Jorge Schrauwen <jorge@blackdot.be> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 usr/src/lib/brand/bhyve/zone/boot.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/lib/brand/bhyve/zone/boot.c b/usr/src/lib/brand/bhyve/zone/boot.c
index 1e2c6771be..f4a3f32d18 100644
--- a/usr/src/lib/brand/bhyve/zone/boot.c
+++ b/usr/src/lib/brand/bhyve/zone/boot.c
@@ -43,12 +43,12 @@
 
 typedef enum {
 	PCI_SLOT_HOSTBRIDGE = 0,	/* Not used here, but reserved */
-	PCI_SLOT_LPC,
-	PCI_SLOT_CD,
+	PCI_SLOT_CD = 3,		/* Windows ahci allows slots 3 - 6 */
 	PCI_SLOT_BOOT_DISK,
 	PCI_SLOT_OTHER_DISKS,
 	PCI_SLOT_NICS,
 	PCI_SLOT_FBUF = 30,
+	PCI_SLOT_LPC = 31,		/* Windows requires lpc in slot 31 */
 } pci_slot_t;
 
 static boolean_t debug;
-- 
2.21.0

